zombienet/tests/0007-full_node_warp_sync.toml

```
@@ -0,0 +1,62 @@
+[relaychain]
+default_image = "{{RELAY_IMAGE}}"
+default_command = "polkadot"
+default_db_snapshot = "https://storage.googleapis.com/zombienet-db-snaps/cumulus/0007-full_node_warp_sync/relaychain-0b03131ee21dd174ea695f9cbfaed84720b742d8.tgz"
+
+chain = "rococo-local"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = true
+
+  [[relaychain.nodes]]
+  name = "bob"
+  validator = true
+
+[[parachains]]
+id = 2000
+cumulus_based = true
+chain_spec_path = "0007-warp-sync-parachain-spec.json"
+add_to_genesis = false
+
+  [[parachains.collators]]
+  name = "charlie"
+  validator = true
+  image = "{{COL_IMAGE}}"
+  command = "polkadot-parachain"
```

```suggestion
  command = "test-parachain"
```
All of these should be test-parachain like the other tests.